numpy/core/_type_aliases.py

```diff
@@ -50,7 +50,9 @@
     "cdouble": "complex128",
     "single": "float32",
     "csingle": "complex64",
-    "half": "float16"
+    "half": "float16",
+    "int_": "long",
+    "uint": "ulong",
```

Comment:
Do we want this the other way around, so that `long` is somehow the "canonical" name? Maybe that's better left to a follow-up PR.

Reply:
Nevermind, I misread what this dictionary meant.